foxy/distribution.yaml

```diff
@@ -3083,6 +3083,24 @@ repositories:
       url: https://github.com/ros-controls/realtime_tools.git
       version: foxy-devel
     status: maintained
+  repository_name:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/lua_vendor.git
+      version: 0.0.1
```

Comment:
@hakuturu583 hmm, I have the impression that using a tag for a `doc` version won't work. @clalancette ?

Reply:
I have the same impression, but I also though that our tests would have pointed that out and failed.  So I'm not sure what is going on.

To be safe, I'll suggest that we do indeed change this to a branch name.